doc/qbk/3.1.parsing.qbk

```diff
@@ -123,4 +123,65 @@ information please consult the full
 [@boost:/libs/system/doc/html/system.html#ref_resultt_e `result`]
 documentation in [@boost:/libs/system/doc/html/system.html Boost.System].
 
+[heading Formatting]
```

Comment:
Why is "Formatting" in the Parsing section?

Reply:
It can be in the Parsing section, another section after "Parsing", or in the Containers "section"? It does involve parsing the template though.

![image](https://user-images.githubusercontent.com/5369819/209869052-865a9069-c53d-4d77-ab14-3a04096dc405.png)

Reply:
it has nothing to do with parsing.

Reply:
put it after Percent Encoding, as Formatting